# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/base/MemoryTelemetry.cpp
# Commit: bba32db4da1e
# Full Hash: bba32db4da1e4096b8286710b7f2a550796a2add
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2021-03-31 21:54:44
# Regressor Bug: 1586236
# File Overlap Count: 1
# Description:
#   Bug 1586236 - Use memory resource notifications to detect low memory scenarios on Windows. r=gsvelto
#   
#   This patch uses the low memory resource notification facility to detect
#   scenarios where physical memory is running low without polling. This is a
#   significant change compared to the previous behavior which measured both
# ==============================================================================

diff -r 1fc94d717bee -r bba32db4da1e xpcom/base/MemoryTelemetry.cpp
--- a/xpcom/base/MemoryTelemetry.cpp	Wed Mar 31 15:13:16 2021 +0000
+++ b/xpcom/base/MemoryTelemetry.cpp	Wed Mar 31 15:18:05 2021 +0000
@@ -257,10 +257,6 @@
          UNITS_BYTES);
   RECORD(MEMORY_STORAGE_SQLITE, StorageSQLite, UNITS_BYTES);
 #ifdef XP_WIN
-  RECORD(LOW_MEMORY_EVENTS_VIRTUAL, LowMemoryEventsVirtual,
-         UNITS_COUNT_CUMULATIVE);
-  RECORD(LOW_MEMORY_EVENTS_COMMIT_SPACE, LowMemoryEventsCommitSpace,
-         UNITS_COUNT_CUMULATIVE);
   RECORD(LOW_MEMORY_EVENTS_PHYSICAL, LowMemoryEventsPhysical,
          UNITS_COUNT_CUMULATIVE);
 #endif